ament_tools/build_types/cmake.py

```diff
@@ -167,12 +177,15 @@ def _common_cmake_on_test(self, context, build_type):
         prefix = self._get_command_prefix('test', context)
         if not IS_WINDOWS:
             if has_make_target(context.build_space, 'test') or context.dry_run:
-                if MAKE_EXECUTABLE is None:
-                    raise VerbExecutionError("Could not find 'make' executable")
-                cmd = prefix + [MAKE_EXECUTABLE, 'test']
-                if 'ARGS' not in os.environ:
-                    cmd.append('ARGS="-V"')
-                yield BuildAction(cmd)
+                if CTEST_EXECUTABLE is None:
+                    raise VerbExecutionError("Could not find 'ctest' executable")
+                # invoke CTest directly in order to pass arguments
+                yield BuildAction(prefix + [
+                    CTEST_EXECUTABLE,
+                    # show all test output
+                    '-V',
+                    '--force-new-ctest-process'] +
+                    context.ctest_args)
```

Comment:
With stuff like http://public.kitware.com/Bug/view.php?id=8438 potentially being added in the feature it could very well be that such things will be backed into the make target and would therefore be bypassed by invoking ctest directly. So I will remove the usage of ctest from this PR.

There is only one questions: how to handle `ARGS` then? I would argue since merging `ARGS` with custom `--ctest-args` is not feasible ignore `ARGS` all together?